.github/workflows/pythonpackage.yml

```diff
@@ -24,8 +24,12 @@ jobs:
         pip install poetry
         poetry config virtualenvs.create false
         poetry install
-    - name: Lint with flake8
+    - name: Lint with autopep8 and flake8
       run: |
+        pip install autopep8
+        autopep8 --in-place --aggressive --aggressive base.py
+        autopep8 --in-place --aggressive --aggressive backend.py
+        autopep8 --in-place --aggressive --aggressive requirements.py
```

Comment:
I'm not sure why we're running autopep8 here. Isn't it a code formatter not a linter?